packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -76,17 +76,20 @@ export class VaButton {
   })
   componentLibraryAnalytics: EventEmitter;
 
-  private handleClick = (e: MouseEvent): void => {
-    // TODO: remove console.log
-    console.log(e);
-    if (this.disableAnalytics) return;
-    this.componentLibraryAnalytics.emit({
-      componentName: 'va-button',
-      action: 'click',
-      details: {
-        // TODO: add analytics event details
-      },
-    });
+  private handleClick = (): void => {
+    if (!this.disableAnalytics) {
+      const detail = {
+        componentName: 'va-button',
+        action: 'click',
+        details: {
+          event: 'cta-button-click',
```

Comment:
What does this do for us?

Reply:
The GA event logs as a button click for `va-button` which they are labeling as a "call to action button click (`cta-button-click`). These are the details I received from the analytics team.

Reply:
Can't that be added in the `vets-website` mapping where it's more specific to the GA implementation?

Reply:
I understand what you mean now. The part `event: 'cta-button-click',` [is in vets-website](https://github.com/department-of-veterans-affairs/vets-website/blob/77e40f5431f228ffba7a159dd377f5a09d3e4d9b/src/platform/site-wide/component-library-analytics-setup.js#L180-L185) already. I'm not sure why it was added here. Removing.